### PR TITLE
Added the hide blog feature

### PR DESCRIPTION
**Pull request by achu-krishna**
Add a new field in the `yourData.json` named `hideBlog` which is a boolean field.

In the `Header/index.js` used logical and operator to hide the navItem

### CODE REVIEW

**Review by chetanverma16 — APPROVED** [review]
Awesome
Looks Great!

**Comment by chetanverma16** [comment]
Go Ahead And merge Away 🚀

**Comment by achu-krishna** [comment]
> Go ahead And merge Away rocket

I can't merge this. I don't have write access, only you can merge

**Comment by Aryan3212** [comment]
Hi nice job! Just a question, if someone were to put in /blog after the URL, wouldn't the route still exist? We are only removing the button here. Maybe we can work on this enhancement later. A very minor issue tbh, great work on the commit 😄

File: yourData.js
Comment (Aryan3212) [inline yourData.js:125]: Can this moved to the top of the json file? It may be more helpful to keep it at the top.

File: components/Header/index.js
Comment (chetanverma16) [reply components/Header/index.js:21]: @achu-krishna

Comment (chetanverma16) [reply components/Header/index.js:21]: @achu-krishna Once you are done with the above resolution we can merge!

Comment (achu-krishna) [reply components/Header/index.js:21]: @chetanverma16 @Aryan3212 Sure I will do it ASAP

**Comment by chetanverma16** [comment]
Yeah,
this is a quick fix as of now to hide the blog.
but we can surely work later on for a better approach to this problem.

@achu-krishna Nice Work!

**Pull request merged**
